src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const listArr = document.querySelectorAll('li');
+
+listArr.forEach(list => {
+  const span = document.createElement('span');
+
+  span.append(list.firstChild);
+
+  list.insertAdjacentElement('afterbegin', span);
+});
+
+document.addEventListener('click', evetio => {
+  if (evetio.target.tagName === 'SPAN') {
+    const eventParent = evetio.target.parentElement;
+
+    eventParent.innerHTML = ``;
```

Comment:
Instead of setting the innerHTML to an empty string, you should toggle the display or hidden property of the child elements.